### ISSUE TITLE
Pagination - Ensure parts of graphical objects essential for understanding content have sufficient contrast - (2036570532)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-design-system#7805]
### Violation: 
 Ensure parts of graphical objects essential for understanding content have sufficient contrast 

![image](https://github.com/Esri/calcite-design-system/assets/5023024/3c78a9b2-5191-4451-8bcd-6275aa14bf5d)

### WCAG Reference: 
- [1.4.11 Non-test Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) 

### Severity: 
 5 

### Media Type: 
 Color and Contrast 

--- 
### Areas for Remediation: 
- Title: Pagination - Ensure parts of graphical objects essential for understanding content have sufficient contrast - (2036570532) 
- Module: Pagination 
**Issue**
There are graphical objects with a color contrast ratio below 3.00:1.

Foreground: #9F9F9F
Background: #F8F8F8
Contrast ratio: 2.5:1

Example includes:
- <
- > (Next, Previous button)

**User Impact**
Users with low vision will have difficulty identifying this content.

 
**Suggestion**
Ensure parts of graphical objects essential for understanding content have sufficient contrast. The required minimum contrast ratio is 3.00:1.

Common examples of qualifying objects include lines in a chart, meaningful icons, and annotations within an image.

Graphics that require particular presentation to preserve their meaning are exempt from this requirement.

To evaluate color contrast, see the Accessible Color Picker extension: https://www.accessibility.dev/

 
--- 
### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by driskull** [comment]
I think this might be the disabled buttons? should they be `aria-hidden`?

**Comment by geospatialem** [comment]
`pagination` uses `--calcite-color-text-3` for its outline, which provides sufficient color contrast in light and dark modes:
- Light mode: [5.09:1](https://webaim.org/resources/contrastchecker/?fcolor=6A6A6A&bcolor=F8F8F8)
- Dark mode: [4.63:1](https://webaim.org/resources/contrastchecker/?fcolor=9F9F9F&bcolor=353535)

![image](https://github.com/Esri/calcite-design-system/assets/5023024/3b6ae510-dd65-419b-bb0b-a177f31fc10d)
![image](https://github.com/Esri/calcite-design-system/assets/5023024/9aacc298-fedb-4683-8065-3f0fc6d2f80c)

Closing as no longer valid as of `2.0.0`
